index.html

```diff
@@ -8,11 +8,16 @@
   <link rel="stylesheet"
   href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
-<body>
-  <header>
-    <nav class="header">
+<body class="container">
+  <header class="head">
+    <nav class="header hide-desktop">
       <img src="assets/images/menu.png" class="menu-btn"">
```

Comment:
- [ ] your hamburger on the mobile version is not placed correctly please kindly fix this design :+1: 
![ham](https://user-images.githubusercontent.com/89055502/236409547-8d0da239-5a46-4d50-b9a2-50fed29d38b6.png)